privaterelay/templates/faq.html

```diff
@@ -136,6 +137,10 @@ <h2 class="faq-headline">{% ftlmsg 'faq-question-unsubscribe-domain-question' %}
               <h2 class="faq-headline">{% ftlmsg 'faq-question-8-question' %}</h2>
               <p class="faq-answer js-set-href">{% ftlmsg 'faq-question-8-answer-html' url='https://www.mozilla.org/privacy/firefox-relay/' attrs='class="text-link" data-url="https://www.mozilla.org/privacy/firefox-relay/"' %}</p>
             </section>
+            <section class="faq" id="faq-data-collection">
```

Comment:
Duplicated ID from preceding question 

```suggestion
            <section class="faq" id="faq-privacy-notice">
```